### DISCUSSION TITLE
Missing data when using companion app

### DISCUSSION THREAD

**Discussion by Stefan-Wolff** [NightscoutFoundation/xDrip#3211] [Q&A]
Hi, im using the companion app as data source for xdrip+. The sugar values from the companion app (CamAPS FX) are transferred correctly. However, the meal times (insulin and carbohydrates) are missing. What can I do to ensure that they are also transferred?

**Comment by Navid200** [comment]
As far as I know, there is no such capability.  The companion app feature relies on notifications from the companion app to gather the data it needs.
There is an enhancement to add backfill for just a few particular apps.
But, as far as I know, there is no enhancement to transfer carbs or insulin.

**Reply by Stefan-Wolff** [reply]
Would it make sense to add this feature?

**Reply by Navid200** [reply]
Definitely!
All developers are encouraged to open PRs (submissions) to enhance xDrip.  They will all be considered.
